load/InputFileset.py

```diff
@@ -64,9 +67,16 @@ def filesetCommandLineParser(config, s3Client, lptsReader):
 		filesetPaths = sys.argv[3:]
 
 		for filesetPath in filesetPaths:
+			stocknumbers = PreValidate.getStockNumbersFromFile(filesetPath, location.replace("s3://", ""))
+			hasStocknumbersFile = True if len(stocknumbers) > 0 else False
 			filesetPath = filesetPath[:-1] if filesetPath.endswith("/") else filesetPath
-			filesetId = filesetPath.split("/")[-1]
-			filesetId = filesetId[:7] + "_" + filesetId[8:] + "-usx" if filesetId[8:10] == "ET" else filesetId
+
+			directory = filesetPath.split("/")[-1]
+			filesetId = directory
+
+			if hasStocknumbersFile == False:
+				filesetId = InputFileset.parseFilesetId(directory)
+
 			(dataList, messages) = PreValidate.validateDBPELT(lptsReader, s3Client, location, filesetId, filesetPath)
```

Comment:
please add the following comment: the variable filesetId refers to a directory name. Originally, the directory name was an actual fileset id. Now, the directory name can be either (a) a fileset id; or (b) the name of a directory which contains a file called stocknumber.txt, which contains one or more stocknumbers associated with the content in that directory

Reply:
> please add the following comment: the variable filesetId refers to a directory name. Originally, the directory name was an actual fileset id. Now, the directory name can be either (a) a fileset id; or (b) the name of a directory which contains a file called stocknumber.txt, which contains one or more stocknumbers associated with the content in that directory

@bradflood, the comment has been added.